src/app/components/acf/acf.html

```diff
@@ -2,8 +2,7 @@
   <div class="form-horizontal" ng-form="vm.queryForm" ng-if="vm.mode === 'select' || vm.mode === 'enter' || vm.mode === 'edit'">
     <div class="panel panel-primary">
       <div class="panel-heading">
-        <h2 class="panel-title" ng-if="vm.mode === 'select' || vm.mode === 'enter'">Login to an Alternate Care Facility (ACF)</h2>
-        <h2 class="panel-title" ng-if="vm.mode === 'enter'">Create a new ACF</h2>
```

Comment:
I'm not sure if this will work. One of the use cases (though not for CALEMSA) is that a new ACF can be created too. In your `/config/local.json` file there's a parameter for `isAcfEditingAllowed` or something like that. If you set that to `true` then the ACF selection page should allow the option for creating a new ACF. I believe that's the `mode === 'enter'` setting, and now it looks like that workflow would have the wrong title